packages/palette/src/elements/Avatar/Avatar.shared.tsx

```diff
@@ -93,11 +92,11 @@ export const BaseAvatar = ({
 }
 
 /** InitialsHolder */
-export const InitialsHolder = styledWrapper(Flex)<{ size: SizeKey }>`
+export const InitialsHolder = styledWrapper(Flex)`
   background-color: ${color("black10")};
-  border-radius: ${props => sizeValue(props.size).diameter};
   text-align: center;
   overflow: hidden;
+  border-radius: 50%;
```

Comment:
So to recap this change, `size` was being passed down to the `InitialsHolder` only to calculate the radius, but because it's just supposed to be round `border-radius: 50%` achieves the same thing thereby making the passing down of size to this component unnecessary?

Reply:
Yeah, it was doing it manually, and size is included with space so it was overwriting it and setting: `width: xs; height: xs`